apisix/utils/log-util.lua

```diff
@@ -96,6 +96,8 @@ local function get_full_log(ngx, conf)
         }
     end
 
+    local latency, upstream_latency, apisix_latency = _M.latency_details_in_ms(ctx)
```

Comment:
Better to refactor the method and avoid calling the method like `= latency_details_in_ms(ctx)`

Reply:
ok, already update.